manifests/r/Roblox/Launcher/1.6.0.31052/Roblox.Launcher.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Compiled and built by thomasthebro1 on GitHub (Thomas Austin-Crowe)
+# Configured for winget developed by Microsoft Corporation
+# Source: https://www.roblox.com/download/client
+
+PackageIdentifier: Roblox.Launcher
+PackageVersion: 1.6.0.31052
+Installers:
+  - Architecture: x86
+    InstallerType: nullsoft
+    InstallerSwitches:
+      Silent: /S
+      SilentWithProgress: /S
```

Comment:
```suggestion
```

Reply:
I forgot to tell that you can remove these lines because `InstallerType` is set to `nullsoft`

Reply:
I'll sort that now. Thanks for the heads up!